test/e2e/staged_updaterun_test.go

```diff
@@ -253,6 +254,403 @@ var _ = Describe("test RP rollout with staged update run", func() {
 			Eventually(rpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update RP %s status as expected", rpName)
 		})
 	})
+
+	Context("Test resource scale out and shrink using pickFixed policy with staged update run", Ordered, func() {
+		var strategy *placementv1beta1.StagedUpdateStrategy
+		updateRunNames := []string{}
+
+		BeforeAll(func() {
+			// Create a test namespace and a configMap inside it on the hub cluster.
+			createWorkResources()
```

Comment:
Let's move creating resources and CRP out as a `BeforeEach(OncePerOrdered)` block as the other tests?